### PR TITLE
Salon tweaks

### PR DESCRIPTION
**Pull request by lowercase-donkey**
Rolling all my tweaks into one commit so I can fix merge issues.  This kiiiiiiinda got out of hand in terms of commits, but I like where it is now.

Shoutout to @kopareigns for helping me solve some problems with variables!

### CODE REVIEW

**Comment by lowercase-donkey** [comment]
While I'm in here:

`<<if ($tannableSkin.indexOf($activeSlave.skin) != -1)>>`
`	<br>&nbsp;&nbsp;&nbsp;&nbsp;`
`	[[Tan|Salon][$activeSlave.skin = "tanned",$cash -= $modCost]]`
`<<elseif ($lightenableSkin.indexOf($activeSlave.skin) != -1)>>`
`	<br>&nbsp;&nbsp;&nbsp;&nbsp;`
`	[[Lighten skin|Salon][$activeSlave.skin = "lightened",$cash -= $modCost]]`
`<</if>>`
`<<if ($bleachableSkin.indexOf($activeSlave.skin) != -1)>>`
`	<br>&nbsp;&nbsp;&nbsp;&nbsp;`
`	[[Bleach skin|Salon][$activeSlave.skin = "pale",$cash -= $modCost]]`
`<</if>>`
`<<if $activeSlave.skin != "black">>`
`	<br>&nbsp;&nbsp;&nbsp;&nbsp;`
`	[[Darken skin|Salon][$activeSlave.skin = "dark",$cash -= $modCost]]`
`<</if>>`

Is there a reason that Tan/Lighten cannot appear at the same time?  Darken and Bleach are separated.

**Comment by lowercase-donkey** [comment]
Hair style descriptions are incredibly frustrating.  The default "her hair is short" works fine but "her hair is curls" is poor.

**Comment by lowercase-donkey** [comment]
I could go with curly instead of curled, but curls is almost impossible to deal with here without using special code, as happens elsewhere.  It seems easier to just get it to match "permed," but curly would be nice.

**Comment by lowercase-donkey** [comment]
Same work on contacts, but players can pick only shape or only color (as before) as well as both now. 

Hello, demonic RED eyes....

File: src/uncategorized/salon.tw [hunk 1]
```diff
@@ -66,47 +66,265 @@ $activeSlave.slaveName is seated in the auto salon. $pronounCap is awaiting your
...
+	| [[Demonic|Salon][$artificialEyeShape = "demonic"]]
+	| [[Hypnotic|Salon][$artificialEyeShape = "hypnotic"]]
+	| [[Heart-shaped|Salon][$artificialEyeShape = "heart-shaped"]]
+		
+	<<if $artificialEyeColor == "" && $artificialEyeShape != "">>
+		<br>&nbsp;&nbsp;&nbsp;&nbsp;	
+		[[Apply|Salon][$activeSlave.eyeColor = ($artificialEyeShape),$cash -= $modCost, $artificialEyeColor = "", $artificialEyeShape = ""]] $artificialEyeShape $artificialEyeColor now? //This will be applied in addition to eyewear choices.//
+	<<elseif $artificialEyeColor != "" || $artificialEyeShape != "">>
+		<br>&nbsp;&nbsp;&nbsp;&nbsp;	
+		[[Apply|Salon][$activeSlave.eyeColor = ($artificialEyeShape + " " + $artificialEyeColor),$cash -= $modCost, $artificialEyeColor = "", $artificialEyeShape = ""]] $artificialEyeShape $artificialEyeColor now? //This will be applied in addition to eyewear choices.//
```
Comment (kopareigns) [inline src/uncategorized/salon.tw]: The only thing I would recommend changing a bit is the wording. Right now the button reads "Apply blue now" or similar. Just add "lenses" after the shape and color. And maybe change "Apply" to "Give her." That way it'll look like "Give her blue lenses now" which actually makes sense.

Comment (lowercase-donkey) [inline src/uncategorized/salon.tw]: I tried it with lenses, but that section of code also has to apply to glass eyes as well.  I couldn't think of a way to use the "give her X" language that would cover both.

Comment (kopareigns) [inline src/uncategorized/salon.tw]: Couldn't you just put it in an additional if statement? E.g.
`<<if $activeSlave.origEye == "none">>glass eyes<<else>>lenses<</if>>`

Also the if statement between lines 79 and 83 can be entirely removed. You've already checked if $activeSlave.eyeColor == "empty" and are in the else portion. So they're guaranteed to be != "empty" already.

**Comment by freecitiesdev** [comment]
>Wow I am dumb.

**Pull request merged**